docs/reference/workflows/overview.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+The SpatialOS GDK for Unity provides functionality to develop games in both of Unity's paradigms:
+
+* Object-oriented: MonoBehaviours and GameObjects
+* Data-oriented: Entity-Component-System (ECS)
+
+You can stick to familiar MonoBehaviour-based development, check out the new ECS workflow, or try a combination of both.
+
+> However you choose to develop your game, it is important to understand what [SpatialOS entities]({{urlRoot}}/reference/glossary#spatialos-entity) are and how they relate to each workflow.
+
+## MonoBehaviour-centric workflow
+
+In the GDK, SpatialOS entities can be linked to GameObjects. [Readers and Writers]({{urlRoot}}/reference/workflows/monobehaviour/interaction/reader-writers/index) allow you to inspect and change the state of components on SpatialOS entities using MonoBehaviours that you add to their linked GameObjects.
+
+You may find that not all SpatialOS entities need to be represented as a GameObjects. The [Game Object Creation Feature Module]({{urlRoot}}/modules/game-object-creation/overview) allows you to customise the GameObject creation and linking process.
```

Comment:
```suggestion
You may find that not all SpatialOS entities need to be represented as GameObjects. The [Game Object Creation Feature Module]({{urlRoot}}/modules/game-object-creation/overview) allows you to customise the GameObject creation and linking process.
```

I would also mention the feature module in the first sentence where you say that entities can be linked to GameObjects to make it clear that you need to use that module to have them linked